Feature/bugfix/talent pool/job listing card display bug #881

Ameer-Alaswad opened the pull request:
## What Github issue does this PR relate to? Insert link.

Ticket [#881](https://github.com/talent-connect/connect/issues/881)

## What should the reviewer know?

- The peragraph which diplays the job title is not hidding the overflowed content, giving overfow:hidden to it or/and it's parent did not solve the issue, this compoent is nested in losts of components and i had to track all it's parents to the find the parents compoents that are responsible for this issue, i found them and gave them overflow:hidden and this has solved the problem. 

- After the fix 

![bug-fixed](https://github.com/talent-connect/connect/assets/17381734/68ade4f0-a687-4f63-aba9-5ab2041b988c)

Responsive

![bug-fixed-responsive](https://github.com/talent-connect/connect/assets/17381734/a73e6feb-009a-420a-a1b3-22241a89f455)

ericbolikowski reviewed (changes requested):
Thank you @Ameer-Alaswad!

The PR is hard to read at the moment - looks like the traces of a prettifier with a different config than we use. Can you please revert those changes, push, then request a review from me?

Ameer-Alaswad commented:
> Thank you @Ameer-Alaswad!
> 
> The PR is hard to read at the moment - looks like the traces of a prettifier with a different config than we use. Can you please revert those changes, push, then request a review from me?

- Hi @ericbolikowski

- Thanks for your feedback! I've addressed the formatting issues in the PR. The changes you noticed were related to formatting discrepancies, specifically extra spaces around brackets. I've reverted those changes and ensured that the code now aligns with our standard formatting configuration.

ericbolikowski reviewed (approved):
Works beautifully! Well done 👏 Approving.

Will merge & deploy right away.
